migrations/00004_create_task_schema.sql

```diff
@@ -0,0 +1,42 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS tasks (
+    id BIGSERIAL PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    description VARCHAR(511) NOT NULL,
+    difficulty SMALLINT NOT NULL,
+    content TEXT NOT NULL,
+    author VARCHAR(63) NOT NULL,
```

Comment:
`author` should refer to the user's ID, instead of their name, so it should be `BIGINT`